apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -554,13 +556,20 @@ impl BridgeQueryPlanner {
                     QueryPlanResult {
                         query_plan: QueryPlan { node: Some(node) },
                         formatted_query_plan,
+                        evaluated_plan_count,
                     },
                 mut usage_reporting,
             } => {
                 if let Some(sig) = operation_signature {
                     usage_reporting.stats_report_key = sig;
                 }
 
+                u64_histogram!(
+                    "apollo.router.query_planning.evaluated_plans",
```

Comment:
would it make sense to put `evaluated_plans` under `plan` separator as well? in the same we have `apollo.router.query_planning.plan.duration`? so `apollo.router.query_planning.plan.evaluated_plans`

Reply:
sure!